From 36d525329c2d7e6e991f805e63d6b8cc8d207f6b Mon Sep 17 00:00:00 2001
Message-ID: <36d525329c2d7e6e991f805e63d6b8cc8d207f6b.1707763691.git.sharlatanus@gmail.com>
In-Reply-To: <e6295f25eabb9fcc8765561825ab7c9b124c1568.1707763691.git.sharlatanus@gmail.com>
References: <e6295f25eabb9fcc8765561825ab7c9b124c1568.1707763691.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 7 Feb 2024 21:08:28 +0000
Subject: [PATCH v2 09/30] gnu: go-github-com-nsqio-go-diskqueue: Move to
 golang-xyz.

* gnu/packages/golang.scm (go-github-com-nsqio-go-diskqueue): Move
from here ...
* gnu/packages/golang-xyz.scm: ... to here.

Change-Id: I2051eb1f8a40eeab9c8cc96c540413f5818dd78e
---
 gnu/packages/golang-xyz.scm | 23 +++++++++++++++++++++++
 gnu/packages/golang.scm     | 23 -----------------------
 2 files changed, 23 insertions(+), 23 deletions(-)

diff --git a/gnu/packages/golang-xyz.scm b/gnu/packages/golang-xyz.scm
index bbf72f1105..1999aab7fb 100644
--- a/gnu/packages/golang-xyz.scm
+++ b/gnu/packages/golang-xyz.scm
@@ -250,6 +250,29 @@ (define-public go-github-com-nats-io-nuid
 very fast, and tries to be entropy pool friendly.")
     (license license:asl2.0)))
 
+(define-public go-github-com-nsqio-go-diskqueue
+  (package
+    (name "go-github-com-nsqio-go-diskqueue")
+    (version "1.1.0")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/nsqio/go-diskqueue")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1hp66hkmfn0nyf3c53a40f94ah11a9rj01r5zp3jph9p54j8rany"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/nsqio/go-diskqueue"))
+    (home-page "https://github.com/nsqio/go-diskqueue")
+    (synopsis "Go package providing a file system backed FIFO queue")
+    (description
+     "The @code{diskqueue} Go package provides a file system backed FIFO
+queue.")
+    (license license:expat)))
+
 (define-public go-github-com-nsqio-go-nsq
   (package
     (name "go-github-com-nsqio-go-nsq")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index b8b5a67613..599fdf9f2d 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -3075,29 +3075,6 @@ (define-public go-github-com-nathan-osman-go-sunrise
 sunset times from geographical coordinates and a date.")
       (license license:expat))))
 
-(define-public go-github-com-nsqio-go-diskqueue
-  (package
-    (name "go-github-com-nsqio-go-diskqueue")
-    (version "1.1.0")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/nsqio/go-diskqueue")
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "1hp66hkmfn0nyf3c53a40f94ah11a9rj01r5zp3jph9p54j8rany"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/nsqio/go-diskqueue"))
-    (home-page "https://github.com/nsqio/go-diskqueue")
-    (synopsis "Go package providing a file system backed FIFO queue")
-    (description
-     "The @code{diskqueue} Go package provides a file system backed FIFO
-queue.")
-    (license license:expat)))
-
 (define-public go-github-com-hebcal-gematriya
   (let ((commit "fe3043f73e415eb82727701d10f2fb40f87675e9")
         (revision "0"))
-- 
2.41.0

